ci/ansible/roles/pulp/tasks/main.yaml

```diff
@@ -1,23 +1,40 @@
 ---
+- block:
+  - name: Open firewall ports (iptables)
+    iptables:
+      action: insert
+      chain: INPUT
+      ctstate: NEW
+      protocol: "{{ item.protocol }}"
+      destination_port: "{{ item.port }}"
+      jump: ACCEPT
+    with_items:
+      - { port: 'http', protocol: 'tcp' }
+      - { port: 'https', protocol: 'tcp' }
+      - { port: 'amqps', protocol: 'tcp' }
+      - { port: 'amqp', protocol: 'tcp' }
 
-- name: Open firewall ports (iptables)
-  iptables:
-    action: insert
-    chain: INPUT
-    ctstate: NEW
-    protocol: "{{ item.protocol }}"
-    destination_port: "{{ item.port }}"
-    jump: ACCEPT
-  with_items:
-    - { port: 'http', protocol: 'tcp' }
-    - { port: 'https', protocol: 'tcp' }
-    - { port: 'amqps', protocol: 'tcp' }
-    - { port: 'amqp', protocol: 'tcp' }
+  - name: Install required packages for RHEL 6
+    action: "{{ ansible_pkg_mgr }} name={{ item }} state=latest"
+    with_items:
+        - libselinux-python
+        - policycoreutils-python
+    notify:
+      - Save IPv4 iptables configuration
+
+  - name: Setup qpid custom repo
+    get_url: url=https://copr.fedorainfracloud.org/coprs/g/qpid/qpid/repo/epel-6/irina-qpid-epel-6.repo dest=/etc/yum.repos.d/copr-qpid.repo
+
+  - name:  Allow Apache to listen on tcp port 5000
+    seport:
+      ports: 5000
+      proto: tcp
+      setype: http_port_t
+      state: present
+    notify:
+      - Save IPv4 iptables configuration
```

Comment:
The way the SELinux settings were set on the RHEL6 machines I was getting from beaker, Apache was not being allowed to bind to the port. This was the solution that I found. 

For reference, this seems to be a issue other RHEL6 users find:
[rhel6 allow httpd to connect to specific port](https://serverfault.com/questions/563872/selinux-allow-httpd-to-connect-to-a-specific-port)

I followed the example on the seport module and it worked, so I was satisfied.
[examples](http://docs.ansible.com/ansible/seport_module.html#examples)

I needed it for RHEL 6, but it was previously unnecessary, so that's why I guarded it in this block.
Honestly, I think the notify statement being there is a mistake. It should be up under the iptables task and it got pushed down there when I was re-organizing the tasks into blocks.